zio-sbt-ecosystem/src/main/scala/zio/sbt/ScalaCompilerSettings.scala

```diff
@@ -190,7 +190,7 @@ trait ScalaCompilerSettings {
             )
           } else Seq.empty
         },
-        Test / parallelExecution := scalaBinaryVersion.value != "3",
+        Test / parallelExecution := scalaBinaryVersion.value != "3", // why not parallel execution for Scala 3?
```

Comment:
Discovered this setting.. why was this set? I would prefer to not have this set.

Reply:
It's good to remove it! However, I don't remember why this line was included. Can you check to ensure it's okay to remove it?